python/cuml/manifold/umap.pyx

```diff
@@ -532,16 +577,29 @@ class UMAP(Base):
             X contains a sample per row.
             Acceptable formats: cuDF DataFrame, NumPy ndarray, Numba device
             ndarray, cuda array interface compliant array like CuPy
+        knn_indices : arra-like (device or host) shape=(n_samples, n_neighbors)
```

Comment:
Same here, about the need to make this as clear as possible for the user. Also, I would be very clear to state that this is specifically assumed to contain the indices / dists for each sample in X.